src/api/src/main/java/edu/miu/cs545/api/dto/PropertyDto.java

```diff
@@ -0,0 +1,17 @@
+
+package edu.miu.cs545.api.dto;
+
+import edu.miu.cs545.api.entity.Address;
+import lombok.Data;
+
+@Data
+public class PropertyDto {
+    Long id;
+    Integer noOfBedrooms;
+    Double noOfBathrooms;
+    Double plotSize;
+    Double price;
+    Double area;
+    String description;
+    Address address;
```

Comment:
Shall we use an AddressDto without using Address directly?

Reply:
There are other places as well. Idea is to use Dto objects whenever expose data outside. So that we have full control.

Reply:
Ok, I will replace with AddressDto